### DISCUSSION TITLE
Prevent conf.yml from being reset after updates?

### DISCUSSION THREAD

**Discussion by fyrfli** [Lissy93/dashy#585] [Help and Support] [answered]
So to prevent those "you are running an outdated version" type messages, I put in watchtower. What happens now is that at least twice a day, my conf.yml gets reset to the default you get upon first install. Is there any way at all to maybe put that file outside the container and maybe link to it? Or will that also rewrite the config upon update check as well?

**Comment by lissy93** [comment, marked as answer]
### Yes, use a volume.

If you're unsure how, then take a look at the [Providing Assets](https://github.com/Lissy93/dashy/blob/master/docs/management.md#providing-assets) section of the management docs, or reference the [Docker manual](https://docs.docker.com/storage/volumes/).

**Reply by lissy93** [reply]
There's lots of other info for beginners in the [Management Docs](https://github.com/Lissy93/dashy/blob/master/docs/management.md) so I recommend taking a look.
Hope that helps :)

**Reply by fyrfli** [reply]
I'll admit I was scanning through it before I wrote my question  ... I really need to unlearn rapid-reading. :( Thanks. Will do.

**Reply by lissy93** [reply]
No problem, feel free to reach out if u need anything else :)
